eng/ci/templates/official/jobs/run-benchmarks.yml

```diff
@@ -9,22 +9,25 @@ parameters:
 - name: additionalCrankArgs
   type: string
   default: ''
+- name: agentName
+  type: string
 
 jobs:
 - job: ${{ parameters.functionAppName }}
 
   variables:
+    agentName: ${{ parameters.agentName }}
```

Comment:
Just so you are aware, you don't need to assign parameters to variables. You can use `${{ parameters.agentName }}` anywhere in this file - including in a pwsh script.

It is fine to use variables if you want to. A small nuance to them though: params are readonly. Variables can be updated between steps. Probably not important here, but something to consider.